contrib/rpm/podman.spec

```diff
@@ -0,0 +1,207 @@
+BuildRequires:  libgpg-error-devel, libseccomp-devel, libselinux-devel, ostree-devel
+BuildRequires:  pkgconfig, runc, skopeo-containers
+BuildRequires:  git
+Requires:       runc, skopeo-containers, conmon
+
+
+%description
+%{summary}
+libpod provides a library for applications looking to use the Container Pod concept popularized by Kubernetes.
+
+%if 0%{?with_devel}
+%package -n libpod-devel
+Summary:       libpod provides a library for applications looking to use the Container Pod concept popularized by Kubernetes
+BuildArch:     noarch
+
+Provides:      golang(%{import_path}/cmd/podman/docker) = %{version}-%{release}
+Provides:      golang(%{import_path}/cmd/podman/formats) = %{version}-%{release}
+Provides:      golang(%{import_path}/libpod) = %{version}-%{release}
+Provides:      golang(%{import_path}/libpod/common) = %{version}-%{release}
+Provides:      golang(%{import_path}/libpod/driver) = %{version}-%{release}
+Provides:      golang(%{import_path}/libpod/layers) = %{version}-%{release}
+Provides:      golang(%{import_path}/pkg/annotations) = %{version}-%{release}
+Provides:      golang(%{import_path}/pkg/chrootuser) = %{version}-%{release}
+Provides:      golang(%{import_path}/pkg/registrar) = %{version}-%{release}
+Provides:      golang(%{import_path}/pkg/storage) = %{version}-%{release}
+Provides:      golang(%{import_path}/utils) = %{version}-%{release}
+
+%description -n  libpod-devel
+%{summary}
+
+This package contains library source intended for
+building other packages which use import path with
+%{import_path} prefix.
+%endif
+
+%if 0%{?with_unit_test} && 0%{?with_devel}
+%package unit-test-devel
+Summary:         Unit tests for %{name} package
+%if 0%{?with_check}
+#Here comes all BuildRequires: PACKAGE the unit tests
+#in %%check section need for running
+%endif
+
+# test subpackage tests code from devel subpackage
+Requires:        %{name}-devel = %{version}-%{release}
+
+%description unit-test-devel
+%{summary}
+libpod provides a library for applications looking to use the Container Pod concept popularized by Kubernetes.
+
+This package contains unit tests for project
+providing packages with %{import_path} prefix.
+%endif
+
+%prep
+%autosetup -Sgit -n %{repo}-%{commit}
+
+%build
+mkdir _build
+pushd _build
+mkdir -p src/%{provider}.%{provider_tld}/%{project}
+ln -s ../../../../ src/%{import_path}
+popd
+ln -s vendor src
+export GOPATH=$(pwd)/_build:$(pwd):$(pwd):%{gopath}
+export BUILDTAGS="selinux seccomp $(hack/btrfs_installed_tag.sh) $(hack/btrfs_tag.sh) $(hack/libdm_tag.sh) containers_image_ostree_stub"
+
+GOPATH=$GOPATH BUILDTAGS=$BUILDTAGS %gobuild -o bin/%{name} %{import_path}/cmd/%{name}
+
+BUILDTAGS=$BUILDTAGS make all
+
+%install
+PREFIX=%{buildroot}/usr
+%make_install PREFIX=${PREFIX}
+%make_install PREFIX=${PREFIX} install.completions
+%make_install PREFIX=${PREFIX} install.cni
+%make_install PREFIX=${PREFIX} install.man
+# We don't want to ship conmon
+rm -f %{buildroot}/usr/libexec/crio/conmon
```

Comment:
Is this right?  I thought conmon was one of the requirements above, is removing it here the right thing?   (Showing my lack of rpm spec knowledge).

Reply:
This is because at the time this spec was being built conmon was being provided by libpod. We have removed conmon, so this code is no longer needed.  Removing.